scripts/tinybird/retrieve_legacy_data_from_circleci.py

```diff
@@ -0,0 +1,240 @@
+import datetime
```

Comment:
To answer the question in the PR description here:
I think it's okay to keep this one-off script in the PR, maybe just add a short header here to indicate that this is just a script to be used if necessary?
Going forward, I think it might also make sense to make the collection scripts independent of the localstack code base. Then we could extract the different metric aggregators (like this and the terraform test suite) to their own repo - containing all the metric aggregation scripts - with their individual actions / workflows being executed in the scope of that repo. (But this is just an idea 😄)